src/common.js

```diff
@@ -284,30 +284,26 @@ function wrap(cmd, fn, options) {
         //    `cp([file1, file2, file3], dest);`
         // equivalent to:
         //    `cp(file1, file2, file3, dest);`
-        args = args.reduce(function(accum, cur) {
-          if (Array.isArray(cur)) {
+        args = args.reduce(function (accum, cur) {
+          if (Array.isArray(cur))
             return accum.concat(cur);
```

Comment:
Nit: Can we either add curly braces here, or have the `return` on the same line as the `if`?